api/routes/userRouter.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const { userController } = require("../controllers");
+
+const router = express.Router();
+
+router.post("/kakao/signIn", userController.kakaoSignIn);
```

Comment:
```suggestion
router.post("/kakao/sign-in", userController.kakaoSignIn);
```
엔드포인트를 설계할 때는 보통 카멜케이스를 쓰지 않습니다. 필요시 하이픈(`-`)을 활용할 수 있도록 해주세요